docs/src/index.md

```diff
@@ -55,6 +55,12 @@ AutoChainRules
 AutoDiffractor
 ```
 
+### Forward, reverse, or sparse mode
```

Comment:
I recommend a separate category altogether, like "Parallel acceleration", outside of the rest of AD backends. This split would make it clearer that we're talking about a different kind of object.

Also, adding "sparse mode" in there makes it sound like an alternative to the `AutoSparse` struct from ADTypes. But from what I understand, Reactant won't be good at unstructured (scalar-level) sparsity, and will mostly benefit from structured (blockwise) sparsity? This distinction might be confusing for users of the ADTypes ecosystem if not explained.

Reply:
I'm very curious then, see https://github.com/EnzymeAD/Reactant.jl/issues/1868